pandas/core/base.py

```diff
@@ -994,16 +1014,24 @@ def min(self):
         >>> idx.min()
         ('a', 1)
         """
+        nv.validate_minmax_axis(axis)
         return nanops.nanmin(self.values)
 
-    def argmin(self, axis=None):
+    def argmin(self, axis=None, skipna=True):
         """
         Return a ndarray of the minimum argument indexer.
 
+        Parameters
+        ----------
+        axis : {None}
+            Dummy argument for consistency with Series
+        skipna : bool, default True
+
         See Also
         --------
         numpy.ndarray.argmin
         """
+        nv.validate_minmax_axis(axis)
```

Comment:
should think about wether we can / should move these checks into the nanops themselves

Reply:
previously discussed, this check is only for 1-d objects where `axis` is a dummy attribute